src/std/text/hex.ss

```diff
@@ -23,39 +23,46 @@
 (def (hex u4)
   (string-ref hexes u4))
 
-(def unhexes
-  (let (ht (make-hash-table-eq))
-    (for-each (cut hash-put! ht <> <>)
-              (string->list "0123456789")
-              (iota 10))
-    (for-each (cut hash-put! ht <> <>)
-              (string->list "abcdef")
-              (iota 6 10))
-    (for-each (cut hash-put! ht <> <>)
-              (string->list "ABCDEF")
-              (iota 6 10))
-    ht))
+(def 0-code (char->integer #\0))
+(def unhexes-len (- (char->integer #\g) 0-code))
+(def unhexes (make-u8vector unhexes-len 255))
+(def (unhex-index c) (##fx- (char->integer c) 0-code))
+(let loop ((i 0))
```

Comment:
newline before the function body please, and also leave an empty line before starting the init loop.